x-pack/metricbeat/module/sql/query/query.go

```diff
@@ -385,3 +390,18 @@ func inferTypeFromMetrics(ms mapstr.M) mapstr.M {
 
 	return ret
 }
+
+// sanitizeError replaces all occurrences of 'sensitive' in err.Error() with "(redacted)"
+func sanitizeError(err error, sensitive string) error {
+	if err == nil {
+		return nil
+	}
+
+	if sensitive == "" {
+		return err
+	}
```

Comment:
I think we could combine these, like so:
```suggestion
	if err == nil || sensitive == "" {
		return err
	}
```